geode-client-native/src/cppcache/impl/CppCacheLibrary.cpp

```diff
@@ -79,7 +84,6 @@ typedef ACE_Singleton<CppCacheLibrary,ACE_Recursive_Thread_Mutex> TheLibrary;
 // internals are prepared. fw_helper framework will handle this.
 CppCacheLibrary* CppCacheLibrary::initLib( void )
 {
-  ACE::init();
```

Comment:
OK it will be interesting to see how the geode native client handles such scenarios of ACE being used elsewhere. Did you see my other comments, especially the one about the "uint8 m_suffixFiller[32];" added to guard against what ACE bug? Any other changes other than just this declaration in header that I'm missing?